Dominus/Core/Engine/Material.cpp

```diff
@@ -200,7 +200,5 @@ std::string Material::getStencilTextureMap() const {
 
 
 Material::~Material() {
-    delete ambientColor;
-    delete diffuseColor;
-    delete specularColor;
+
```

Comment:
Ownership in this class needs to be review. Do other objects share this pointers with it?